netebpfext/net_ebpf_ext_drv.c

```diff
@@ -32,58 +32,61 @@ WDF based driver that does the following:
 #define NET_EBPF_EXT_DEVICE_NAME L"\\Device\\NetEbpfExt"
 
 // Driver global variables
-static DEVICE_OBJECT* _net_ebpf_ext_driver_device_object;
+static WDFDEVICE _net_ebpf_ext_device = NULL;
+static DEVICE_OBJECT* _net_ebpf_ext_driver_device_object = NULL;
 static BOOLEAN _net_ebpf_ext_driver_unloading_flag = FALSE;
 
 //
 // Pre-Declarations
 //
 DRIVER_INITIALIZE DriverEntry;
 
-static _Function_class_(EVT_WDF_DRIVER_UNLOAD) _IRQL_requires_same_
-    _IRQL_requires_max_(PASSIVE_LEVEL) void __net_ebpf_ext_driver_unload(_In_ WDFDRIVER driver_object)
+static void
+__net_ebpf_ext_driver_uninitialize_objects()
```

Comment:
```suggestion
_net_ebpf_ext_driver_uninitialize_objects()
```
Only one leading underscore.  Leading double underscores are reserved by the compiler.

https://en.cppreference.com/w/c/language/identifier#Reserved_identifiers
"Note: in C++, identifiers with a double underscore anywhere are reserved everywhere; in C, only the ones that begin with a double underscore are reserved."